Scroller/tests/EditableWrapper-specs.js

```diff
@@ -0,0 +1,104 @@
+import '@testing-library/jest-dom';
+import {render, fireEvent, screen} from '@testing-library/react';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import Button from '../../Button';
+import IconItem from '../../IconItem';
+import Scroller from '../Scroller';
+import css from '../Scroller.module.less';
```

Comment:
Please remove this line. It is not appropriate to import this less file.

Reply:
I accepted your suggestion.